src/lib/y2partitioner/setup_errors_presenter.rb

```diff
@@ -44,37 +44,51 @@ def initialize(setup_checker)
     #
     # @return [String]
     def to_html
-      errors = [boot_errors_html, product_errors_html].compact
-      return "" if errors.empty?
+      errors = errors_html
+      return errors if errors
 
-      errors.join(Yast::HTML.Newline)
+      warnings = [boot_warnings_html, product_warnings_html].compact
+      return "" if warnings.empty?
+
+      warnings.join(Yast::HTML.Newline)
     end
 
   private
 
     # @return [SetupChecker] checker for the current setup
     attr_reader :setup_checker
 
-    # HTML representation for boot errors
+    # HTML representation for boot warnings
     #
-    # @return [String, nil] nil if there is no boot error
-    def boot_errors_html
-      errors = setup_checker.boot_errors
+    # @return [String, nil] nil if there is no boot warning
+    def boot_warnings_html
+      warnings = setup_checker.boot_warnings
       # TRANSLATORS
       header = _("The system might not be able to boot:\n")
 
-      errors_html(header, errors)
+      create_html(header, warnings)
     end
 
-    # HTML representation for mandatory product errors
+    # HTML representation for mandatory product warnings
     #
-    # @return [String, nil] nil if there is no product error
-    def product_errors_html
-      errors = setup_checker.product_errors
+    # @return [String, nil] nil if there is no product warning
+    def product_warnings_html
+      warnings = setup_checker.product_warnings
+      # TRANSLATORS
+      header = _("The system could not work properly because the following warnings were found:\n")
```

Comment:
NP: I would avoid the word `warnings` there. An option could be: `The system could not work properly because the following issues were found:`

Reply:
makes sense. Maybe we should also mention that it is product specific requirements?

Reply:
Yes, it makes sense.